### PR TITLE
Fixes starlight runtime

### PR DESCRIPTION
**Pull request by ghost**
This fixes the starlight runtime and makes it on by default. The issue was turfs aren't initialized at all so I added a line in lighting subsystem that calls update_starlight for space turfs at roundstart, after configs are loaded.

### CODE REVIEW

**Comment by Razharas** [comment]
Man thats one big fucking loop
You sure it doesnt hang/crash/runtime with infinite loop?

**Comment by paprka** [comment]
Fixed.

File: code/controllers/subsystem/lighting.dm [hunk 1]
```diff
@@ -89,6 +89,10 @@ var/datum/subsystem/lighting/SSlighting
 	else
 		changed_turfs.Cut()
 
+	if(config.starlight)
+		for(var/turf/space/S in world)
```
Comment (Iamgoofball) [inline code/controllers/subsystem/lighting.dm]: Why not make this a bit faster by making new space tiles add themselves to a "space_turfs" list and have this loop that list

Comment (paprka) [inline code/controllers/subsystem/lighting.dm]: It's perfectly fine imo

**Comment by paprka** [comment]
This needs to get merged like now

**Pull request merged**